chart/Chart.yaml

```diff
@@ -1,20 +1,5 @@
 apiVersion: v2
 name: uds-dev-stack
 description: UDS dev stack chart
-
-# A chart can be either an 'application' or a 'library' chart.
-#
-# Application charts are a collection of templates that can be packaged into versioned archives
-# to be deployed.
-#
-# Library charts provide useful utilities or functions for the chart developer. They're included as
-# a dependency of application charts to inject those utilities and functions into the rendering
-# pipeline. Library charts do not define any templates and therefore cannot be deployed.
 type: application
-
-# This is the chart version. This version number should be incremented each time you make changes
-# to the chart and its templates, including the app version.
-# Versions are expected to follow Semantic Versioning (https://semver.org/)
-# x-release-please-start-version
-version: 0.9.0
-# x-release-please-end
```

Comment:
Discussed offline but the removal of these comments is not a concern as release-please will continue to update the version regardless.